Zebble/Android/Controls/AndroidTextInput.cs

```diff
@@ -114,6 +118,8 @@ bool IsDead(out TextInput view)
         {
             view = View;
             if (view?.IsDisposing == true) view = null;
+            if (view is null && autoScrollUtility != null)
```

Comment:
@Ali-YousefiTelori I think, it is better to find the main problem of the keyboard instead of adding some extra scroll to it. Please check the problem on different devices and find why the auto-scroll of the keyboard does not set by the operating system.